AppBuilder/platform/ABModelApi.js

```diff
@@ -0,0 +1,69 @@
+//
+// ABModelAPI
+//
+// Represents the Data interface for an ABObjectQuery data.
+
+const ABModel = require("./ABModel");
+
+module.exports = class ABModelAPI extends ABModel {
+   ///
+   /// Instance Methods
+   ///
+
+   /**
+    * @method batchCreate
+    * update model values on the server.
+    */
+   batchCreate(values) {
+      var error = new Error(
+         "ABObjectQuery.ABModelAPI.batchCreate() should not be called."
+      );
+      return Promise.reject(error);
+   }
+
+   /**
+    * @method create
+    * update model values on the server.
+    */
+   create(values) {
+      var error = new Error(
+         "ABObjectQuery.ABModelAPI.create() should not be called."
```

Comment:
Long term, I think we will implement some of these methods. Maybe a better warning is "not implemented."